llvm/lib/Support/CommandLine.cpp

```diff
@@ -1537,6 +1537,23 @@ bool CommandLineParser::ParseCommandLineOptions(int argc,
   GlobalParser->ActiveSubCommand = ChosenSubCommand;
 
   assert(ChosenSubCommand);
+
+#ifdef SYCLomatic_CUSTOMIZATION
+#ifndef _WIN32
+  if (std::string(argv[FirstArg]) == "--intercept-build") {
```

Comment:
how about make "-intercept-build" and "intercept-build" also works.

Reply:
Modified and tested to make `-intercept-build` and `intercept-build` to also work. Done.